_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,144 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Forseti Notifier can send a variety of notifications to alert you
+of Forseti events. You can configure notifications to send to different
+channels and in different formats.
+
+### Notification Types
+
+  1. Inventory summary
+  1. Scanner summary
+  1. Violations
+  1. Cloud Security Command Center (Cloud SCC) findings
+  
+### Notification Channels
+
+  1. Email
+  1. Slack
+  1. Cloud Storage
+
+### Notification Formats
+
+  1. Human-readable data: .csv
+  1. Structured data: JSON
+
+## Configuring Notifier
+
+### Inventory Summary (TBD)
+
+### Scanner Summary (TBD)
+
+### Violation Notifications
+
+To select how you want Notifier to send violation notifications,
+follow the steps below:
+
+1. Open `forseti-security/configs/forseti_conf.yaml`.
+1. Navigate to the `notifier` > `resources` section.
+
+The following options are available, on a per resource basis. You can use
+any combination of notifiers for each resource.
+
+* `should_notify`: Controls whether violation for each resource should be sent.
+  `true` enables the notification, and `false` disables the notification.
+
+* `name`: The name of the notifier you want for each resource.  You can specify
+  multiple notifiers for each resource.
+
+  The name must match the actual module name for each notifier in 
+  `forseti-security/google/cloud/forseti/notifier/notifiers`,
+  such as `email_violations.py`
+
+  These notifiers can be:
+  1. `email_violations`
+  This emails all the violation data via SendGrid.  Multiple email recipients are
+  delimited by comma.
+
+  1. `slack_webhook`
+  This sends individual violations to a Slack channel via a Slack webhook.
+  Configure a webhook in your organization's Slack settings and set the `webhook_url`.
+
+  1. `gcs_violations`
+  This uploads all the violation data to a Cloud Storage bucket.
+
+* `data_format`: Either `csv` (default) or `JSON`.
+  Slack only support json type.
+
+The following example shows how to update a `.yaml` file to add email, Slack,
+and Cloud Storage notifier for Cloud SQL violations:
+
+```yaml
+notifier:
+    resources:
+        - resource: cloudsql_acl_violations
+          should_notify: true
+          notifiers:
+             - name: gcs_violations_pipeline
+               configuration:
+                 gcs_path: gs://path_to_foo_bucket
+             - name: email_violations_pipeline
+               configuration:
+                 sendgrid_api_key: foobar_key
+                 sender: forseti-notify@forsetisecurity.org
+                 recipient: foo@gmail.com,bar@gmail.com,baz@gmail.com
+             - name: slack_webhook_pipeline
+               configuration:
+                 webhook_url: https://hooks.slack.com/services/foobar
+```
+
+### Cloud SCC Findings
+
+Forseti violations can be output for integration with
```

Comment:
More simply

`Violations can be shared with Google Cloud Security Command Center.`